src/util/channel_layout.rs

```diff
@@ -2,7 +2,12 @@ use libc::c_ulonglong;
 
 use crate::ffi::*;
 
+#[cfg(feature = "serde")]
+use serde::{Serialize, Deserialize};
```

Comment:
I think it's cleaner with fully qualifying the derives instead of importing the macros.